adafruit_azureiot/quote.py

```diff
@@ -128,7 +128,7 @@ class Quoter(defaultdict):
     # of cached keys don't call Python code at all).
     def __init__(self, safe):
         """safe: bytes object."""
-        super(Quoter, self).__init__()
```

Comment:
Why are you removing `Quoter` from the superclass?

Reply:
Again, this was to appease the linter, it wanted the super class calls to use python 3 syntax

Reply:
kk - np, this shouldn't effect functionality, I've converted super() calls to Python3 syntax before.